host-configs/hc-butte-blueos_3_ppc64le_ib_p9-xl@beta2019.06.20-cuda-7ymlfxgrfplb5tztddd6ta46zc2iypwg.cmake

```diff
@@ -0,0 +1,41 @@
+###################
+# Generated host-config - Edit at own risk!
```

Comment:
Perhaps this was already discussed somewhere previously, but why are generated files being checked in to the repo?

Reply:
See PR comment: We could now drop them :)

At first, generating them was costly in CI (because starting from scratch for each job meant downloading Spack and configuring it). I optimized the mechanism so that it’s now much faster in CI.

Spack is still downloaded each time, but this clone has been optimized (single branch, single commit).

Reply:
Great to hear.  Let's drop them.

Reply:
Done.

Reminder: old host-config files are still used in Bamboo, so I left them here for now.